.github/workflows/ci.yaml

```diff
@@ -2,8 +2,12 @@ name: Pycloudlib CI
 
 on: [pull_request]
 
+concurrency:
+  group: "ci-${{ github.workflow }}-${{ github.ref }}"
+  cancel-in-progress: true
```

Comment:
This is 'safe' as far as instance cleanup mid a cancelation request because our integration tests already registered a
```
finally:
   instance.delete()
```
at launch time right? So we don't risk leaving unterminated instances after a cancelation event.

Reply:
That's my understanding.